gluten-data/src/main/scala/org/apache/spark/sql/utils/SparkSchemaUtil.scala

```diff
@@ -37,6 +37,16 @@ object SparkSchemaUtil {
     SparkArrowUtil.toArrowSchema(schema, timeZoneId)
   }
 
+  def checkSchema(schema: StructType): Boolean = {
+    try {
+      SparkSchemaUtil.toArrowSchema(schema)
+      true
+    } catch {
+      case _: Exception =>
+        false
+    }
+  }
```

Comment:
What's the error you have seen then decide to catch?

Reply:
I find here will throw exception, https://github.com/apache/incubator-gluten/blob/main/gluten-data/src/main/scala/org/apache/spark/sql/utils/SparkArrowUtil.scala#L54

But it will not run to here actually, I will drop the change.